server/services/notify/notifysubscriptions/diff2markdown.go

```diff
@@ -0,0 +1,181 @@
+package notifysubscriptions
+
+import (
+	"strings"
+
+	"github.com/sergi/go-diff/diffmatchpatch"
+
+	"github.com/mattermost/mattermost-server/v6/shared/mlog"
+)
+
+func diff2Markdown(oldText string, newText string, logger *mlog.Logger) string {
```

Comment:
I don't know if the name is a bit misleading here, I would expect the function with this name to get a "diff" and generate a "markdown", not to generate a diff  and return it in markdown. Maybe I would name it as `generateMarkdownDiff` or `getMarkdownDiff` or something like that.

Reply:
Fixed via https://github.com/mattermost/focalboard/pull/2037/commits/39420ad6d53efe4e7377afa5b197b8fdfbcf3526